Tasks/Maven/maventask.ts

```diff
@@ -258,6 +265,35 @@ mvnv.exec()
     console.error("SonarQube analysis failed");
     sqRunFailed = true;
 })
+.then(function (code) { // Pick up code analysis file(s)
+    var analysisResults: ar.AnalysisResult[] = [];
```

Comment:
The orchestration is rather complex so please put all your code inside a separate function and call it from here.